src/program/lwaftr/setup.lua

```diff
@@ -527,6 +527,15 @@ function reconfigurable(scheduling, f, graph, conf, ...)
    local function setup_fn(conf)
       local graph = config.new()
       f(graph, conf, unpack(args))
+      local name = graph.apps.lwaftr.arg.softwire_config.name
```

Comment:
likewise.  But I see what you were trying to do, to gracefully handle an failed name claim; but we can handle that by instead putting the pcall around the setup_fn call in the leader.  WDYT?